VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+When a validator is selected to propose the next block, it must include at least +2/3 precommits for the previous block in the form of validators signatures. However, there is an incentive to include more than 2/3 precommits in the form of a bonus. The bonus is linear: it ranges from 1% if the proposer includes 2/3rd precommits (minimum for the block to be valid) to 5% if the proposer includes 100% precommits. Of course the proposer should not wait too long or other validator may timeout and move on to the next proposer. As such, validators have to find a balance between wait time to get the most signatures and risk of losing out on proposing the next block. This mechanism aims to incentivize non-empty block proposals, better networking between validators as well as mitigating censorship.
+
+
+Let’s take a concrete example to illustrate the aforementioned concept. In this example, there are 10 validators with equal stake. Each of them applies a 1% commission and has 20% of self-bonded Atoms. We consider a successful block collects a total of 1025.51020408 Atoms in fees.
+
+First, a 2% tax is applied.
+* 2% * 1025.51020408 = 20.51020408 Atoms go to the reserve pool
+
+1005 Atoms now remain. Let’s assume that the proposer included 100% of the signatures in its block. It thus obtains the full bonus of 5%.
+
+We have to solve this simple equation to find the reward R for each validator:
+
+`9*R + R + R*5% = 1005 ⇔ R = 1005/10.05 = 100`
+
+* For the proposer validator
+  * The pool obtains R + R*5% = 105 Atoms
+  * Commission = 105*80%*1% = 0.84 Atoms
+  * Validator reward = 100 * 20% + Commission = 21.84 Atoms
+  * Delegators rewards = 105 * 80% - Commission = 83.16 Atoms (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+* For each non-proposer validator
+  * The pool obtains R = 100 Atoms
+  * Commission = 100*80%*1% = 0.8 Atoms
+  * Validator reward = 100 * 20% + Commission = 20.8 Atoms
+  * Delegators rewards = 100 * 80% - Commission = 79.2 Atoms (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+
+### What are the slashing conditions?
+
+If a validator misbehaves, its bonded stake along with its delegator’s stake and will be slashed. The severity of the punishment depends on the type of fault. There are 3 main faults that can result in loss of funds for a validator and its delegators :
+
+* **Double signing:** If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A
+* **Unavailability:** If a validator’s signature has not been included in the last X blocks, the validator will get slashed by a marginal amount proportional to X. If X is above a certain limit Y, then the validator will get unbonded
+* **Non-voting:** If a validator did not vote on a proposal and once the fault is reported by a someone, its stake will receive a minor slash. 
```

Comment:
The non-voting should be automatic in the same way that unavailability is automatic.

Reply:
Okay, fair enough